client/src/stylesheets/components/_Tools.scss

```diff
@@ -0,0 +1,57 @@
+.tool-panel-wrapper {
+  .tool-panel {
+    margin-bottom: 2em;
+    background-color: $main-blue;
+    .panel-body {
+      ul {
+        list-style-type: square;
+        margin-left: 1em;
+        color: $main-blue;
+      }
+      li {
+        text-align: left;
+        font-size: 88%;
+      }
+      background-color: white;
+      a {
+        color: $main-blue;
+        &:hover,
+        :focus {
+          color: $hovergreen;
+          outline: 0 !important;
+          outline-color: transparent !important;
+        }
+        &:active {
+          color: darken($hovergreen, 5%);
+          outline: 0 !important;
+          outline-color: transparent !important;
+        }
+      }
+    }
+
+    text-align: center;
+    a {
+      text-decoration: none;
+    }
+  }
+  .tool-panel-title {
+    font-size: 20px;
+
+    a {
+      color: white;
+      text-decoration: none;
+    }
+  }
+}
+
+@media (max-width: 992px) {
```

Comment:
what size screen is this 992px?  pull this into _global.scss and give it an identifying label

$tablet-phone: 'only screen and (max-width : 992px)';

Reply:
Pulled this media query into the CollapsePanel scss but did not make the constant.